spring-boot-project/spring-boot-test-autoconfigure/src/test/java/org/springframework/boot/test/autoconfigure/jdbc/TestDatabaseAutoConfigurationTests.java

```diff
@@ -60,6 +60,29 @@ void replaceWithUniqueDatabase() {
 		});
 	}
 
+	@Test
+	void replaceAnyWithCustomUrl() {
+		// There is a user configured HSQL we will replace with H2 using a custom url
```

Comment:
We don't put comment in our test but rather structure our test method name with additional context. Can you please remove those comments?

Reply:
I copied that from another existing test but i agree and will rename and remove.